target/linux/ramips/base-files/lib/upgrade/platform.sh

```diff
@@ -290,6 +290,8 @@ platform_check_image() {
 		;;
 	hc5962|\
 	mir3g|\
+	r6220b|\
+	r6220a|\	
```

Comment:
This line is incorrect - there is a character after the backslash which has to be removed. It results in a non-working sysupgrade script. As this commit is not accepted it is just a reminder for anyone who wants to compile it for him/herself.